README.md

```diff
@@ -197,6 +197,8 @@ perf | Exposes perf based metrics (Warning: Metrics are dependent on kernel conf
 processes | Exposes aggregate process statistics from `/proc`. | Linux
 qdisc | Exposes [queuing discipline](https://en.wikipedia.org/wiki/Network_scheduler#Linux_kernel) statistics | Linux
 slabinfo | Exposes slab statistics from `/proc/slabinfo`. Note that permission of `/proc/slabinfo` is usually 0400, so set it appropriately. | Linux
+softirqs | Exposes detailed softirq statistics from `/proc/softirqs`. | Linux
+supervisord | Exposes service status from [supervisord](http://supervisord.org/). | _any_
```

Comment:
You still have spurious merge conflict changes here. Please clean up your git history.

Reply:
Hang fire on re-reviewing, just fixing up some more stuff